src/commands/snapshot/compare.ts

```diff
@@ -1,183 +1,194 @@
-      const updatedPropertyNames = updated.map(update => update.name)
-      const addedProperty = _.difference(updatedPropertyNames, initial)
-      const removedProperty = _.difference(initial, updatedPropertyNames)
-      const changedProperty: Change[] = []
-
-      for (const update of updated) {
-        if (addedProperty.includes(update.name)) {
-          update.added = true
-          changedProperty.push(update)
-        }
+  public diffCommandProperty(
+    initial: string[],
+    updated: Change[],
+  ): {addedProperty: string[]; changedProperty: Change[]; removedProperty: string[]; updated: Change[]} {
+    const updatedPropertyNames = updated.map((update) => update.name)
+    const addedProperty = difference(updatedPropertyNames, initial)
+    const removedProperty = difference(initial, updatedPropertyNames)
+    const changedProperty: Change[] = []
+
+    for (const update of updated) {
+      if (addedProperty.includes(update.name)) {
+        update.added = true
+        changedProperty.push(update)
       }
-
-      for (const remove of removedProperty) {
-        changedProperty.push({name: remove, removed: true})
-        // The removed flags in not included in the updated flags, but we want it to
-        // so it shows removed.
-        updated.push({name: remove, removed: true})
-      }
-
-      return {addedProperty, removedProperty, updated, changedProperty}
     }
 
-    get changed(): CommandChange[] {
-      return this.commands.map(command => {
-        return {
-          name: command.id,
-          plugin: command.pluginName || '',
-          flags: Object.entries(command.flags).map(flagName => ({name: flagName[0]})),
-          alias: command.aliases.map(alias => ({name: alias})),
-          // we can assert as string because of the filter
-          chars: Object.values(command.flags).map(flag => flag.char).filter(char => typeof char === 'string').map(char => ({name: char as string})),
-        }
-      })
+    for (const remove of removedProperty) {
+      changedProperty.push({name: remove, removed: true})
+      // The removed flags in not included in the updated flags, but we want it to
+      // so it shows removed.
+      updated.push({name: remove, removed: true})
     }
 
-    public async run(): Promise<CompareResponse> {
-      const {flags} = await this.parse(Compare)
-      const oldCommands = JSON.parse(fs.readFileSync(flags.filepath).toString('utf8')) as SnapshotEntry[]
-      const newCommands = this.changed
-      return this.compareSnapshot(oldCommands, newCommands)
-    }
+    return {addedProperty, changedProperty, removedProperty, updated}
+  }
+
+  public async run(): Promise<CompareResponse> {
+    const {flags} = await this.parse(Compare)
+    const oldCommands = JSON.parse(fs.readFileSync(flags.filepath).toString('utf8')) as SnapshotEntry[]
```

Comment:
I know I'm guilty for writing some of these, but we can change to `fs.readFileSync(flags..., 'utf-8')` instead of using `.toString` - but that's a NIT